unittest/dynamics.cpp

```diff
@@ -79,7 +79,16 @@ BOOST_AUTO_TEST_CASE ( test_FD )
   
   Eigen::VectorXd dynamics_residual (data.M * data.ddq + data.nle - tau - J.transpose()*data.lambda_c);
   BOOST_CHECK(dynamics_residual.norm() <= 1e-12);
-  
+
```

Comment:
Can you a dedicated test for this new feature inside this file?

Reply:
I would be reusing 99% of the same code as in forwarddynamics. Since this is only a getter function (no matter how many computations are needed), I think this should remain with the function which does the actual computations.

Reply:
I understand your point, but I really prefer to have a different test for different functions, otherwise, it will become a mess.
In addition, if the content of forwardDynamics is changing (that will happen next week), then you won't be able to really say that the two codes are identical.

Reply:
Done.